src/BrowserApp.tsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import { I18nextProvider } from 'react-i18next';
+import { getDefaultMiddleware } from 'redux-starter-kit';
+import { USER_FOUND } from 'redux-oidc';
 
 import App from './App';
 import i18n from './i18n/i18nInit';
+import createStore from './redux/store';
+
+const store = createStore({
```

Comment:
I think you should bring this part to the store file,  instead defining it here and pass as prop.